src/components/BShuttle/BShuttle.vue

```diff
@@ -0,0 +1,404 @@
+<template>
+  <div class='shuttle'>
+
+    <div class='available'>
+      <label id='available-label'>{{ availableLabel }}</label>
```

Comment:
This `id` won't be unique on the page if there are 2 shuttles. There's a vue attribute you can use, I think, that's a unique identifier we could append to this. https://stackoverflow.com/questions/34950867/vue-js-how-to-set-a-unique-id-for-each-component-instance

Reply:
👍